### PR TITLE
[Tooling] Call `use-bot-for-git` script directly

### PR DESCRIPTION
**Pull request by iangmaia**
Given `use-bot-for-git` already sets up the Git user according to the bot, we can remove this setup from here.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is not assigned to a milestone.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .buildkite/release-pipelines/complete-code-freeze.yml [hunk 1]
```diff
@@ -15,7 +15,8 @@ steps:
     key: complete_code_freeze
     plugins: [$CI_TOOLKIT_PLUGIN]
     command: |
-      source .buildkite/commands/configure-for-release.sh
+      echo '--- :robot_face: Use bot for git operations'
```
Comment (mokagio) [inline .buildkite/release-pipelines/complete-code-freeze.yml]:
```suggestion
      echo '--- :robot_face: Use bot for Git operations'
```

File: .buildkite/release-pipelines/finalize-release.yml [hunk 2]
```diff
@@ -13,7 +13,8 @@ steps:
   - label: Finalize Release
     plugins: [$CI_TOOLKIT_PLUGIN]
     command: |
-      source .buildkite/commands/configure-for-release.sh
+      echo '--- :robot_face: Use bot for git operations'
```
Comment (mokagio) [inline .buildkite/release-pipelines/finalize-release.yml]:
```suggestion
      echo '--- :robot_face: Use bot for Git operations'
```

File: .buildkite/release-pipelines/new-beta-release.yml [hunk 3]
```diff
@@ -17,7 +17,8 @@ steps:
   - label: New Beta Release
     plugins: [$CI_TOOLKIT_PLUGIN]
     command: |
-      source .buildkite/commands/configure-for-release.sh
+      echo '--- :robot_face: Use bot for git operations'
```
Comment (mokagio) [inline .buildkite/release-pipelines/new-beta-release.yml]:
```suggestion
      echo '--- :robot_face: Use bot for Git operations'
```

File: .buildkite/release-pipelines/publish-release.yml [hunk 4]
```diff
@@ -13,7 +13,8 @@ steps:
   - label: Publish Release
     plugins: [$CI_TOOLKIT_PLUGIN]
     command: |
-      source .buildkite/commands/configure-for-release.sh
+      echo '--- :robot_face: Use bot for git operations'
```
Comment (mokagio) [inline .buildkite/release-pipelines/publish-release.yml]:
```suggestion
      echo '--- :robot_face: Use bot for Git operations'
```

File: .buildkite/release-pipelines/start-code-freeze.yml [hunk 5]
```diff
@@ -14,7 +14,8 @@ steps:
     plugins:
       - $CI_TOOLKIT_PLUGIN
     command: |
-      source .buildkite/commands/configure-for-release.sh
+      echo '--- :robot_face: Use bot for git operations'
```
Comment (mokagio) [inline .buildkite/release-pipelines/start-code-freeze.yml]:
```suggestion
      echo '--- :robot_face: Use bot for Git operations'
```

File: .buildkite/release-pipelines/update-metadata-on-app-store-connect.yml [hunk 6]
```diff
@@ -13,7 +13,8 @@ steps:
   - label: Finalize Release
     plugins: [$CI_TOOLKIT_PLUGIN]
     command: |
-      source .buildkite/commands/configure-for-release.sh
+      echo '--- :robot_face: Use bot for git operations'
```
Comment (mokagio) [inline .buildkite/release-pipelines/update-metadata-on-app-store-connect.yml]:
```suggestion
      echo '--- :robot_face: Use bot for Git operations'
```

**Review by mokagio — APPROVED** [review]
Thanks @iangmaia 

I've left a formatting nitpick, Git vs git, which I'll apply. I'll also merge this into `release/4.55` so we can test it tomorrow as part of the release finalization

**Pull request merged**